features/bootstrap/FeatureContext.php

```diff
@@ -178,6 +178,9 @@ public static function create_cache_dir() {
 	 * @param array $parameters context parameters (set them up through behat.yml)
 	 */
 	public function __construct( array $parameters ) {
+		if ( getenv( 'RESTFUL_TEST_DBHOST' ) ) {
```

Comment:
> Should then we make the sister code in restful also use WP_CLI_TEST_DBHOST?

Yes, although RESTful CLI will be affected by https://github.com/wp-cli/wp-cli/issues/3826 in that the test suite will be a Composer dependency, instead of copied into place.